proxy-node-gateway/src/main/java/uk/gov/ida/notification/session/SessionKeys.java

```diff
@@ -1,9 +1,5 @@
 package uk.gov.ida.notification.session;
 
 public interface SessionKeys {
-    String SESSION_KEY_EIDAS_REQUEST_ID = "eidas_request_id";
-    String SESSION_KEY_EIDAS_CONNECTOR_PUBLIC_CERT = "eidas_connector_public_key";
-    String SESSION_KEY_EIDAS_DESTINATION = "eidas_destination";
-    String SESSION_KEY_EIDAS_RELAY_STATE = "eidas_relay_state";
-    String SESSION_KEY_HUB_REQUEST_ID = "hub_request_id";
+    String SESSION_KEY_SESSION_DATA = "session-data";
```

Comment:
`SESSION_KEY_` is in the interface name is there value to having it repeated?

Reply:
I think so. Once it's been imported you don't see the interface name unless you look at the imports. Having `SESSION_KEY_` prefixed makes it more clear. Just having a constant show up called `SESSION_DATA` might be confusing.

Reply:
Cool 👍